.github/workflows/ci_linux.yml

```diff
@@ -0,0 +1,45 @@
+name: Linux Build and Test
+
+on:
+  workflow_call:
+    inputs:
+      amdgpu_families:
+        type: string
+      artifact_run_id:
+        type: string
+      test_runs_on:
+        type: string
+      linux_use_prebuilt_artifacts:
+        type: string
+
+jobs:
+  build_linux_packages:
+    name: Build
+    if: ${{ inputs.linux_use_prebuilt_artifacts == 'false' }}
+    uses: ./.github/workflows/build_linux_packages.yml
+    with:
+      amdgpu_families: ${{ inputs.amdgpu_families }}
+    permissions:
+      id-token: write
```

Comment:
I am not sure how the repository defaults are set but I guess we might need `contents: read` as well? If that's the case lets please explicitly set this here.